global.json

```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
-    "version": "8.0.0-rc.2.23479.6",
-    "allowPrerelease": true,
+    "version": "8.0.0",
```

Comment:
That's incorrect, should be `8.0.100`.

`8.0.0` is the runtime version.

Reply:
Huh, was it incorrect all the time?

Reply:
Technically yes but 100 > 0 so it worked anyway :shrug: